examples/Example/View/Layout.hs

```diff
@@ -65,7 +65,7 @@ sourceLink :: Path -> View c ()
 sourceLink p =
   link sourceUrl "View Source" ~ Style.link
  where
-  sourceUrlBase = [uri|https://github.com/seanhess/hyperbole/blob/0.4/example/|]
+  sourceUrlBase = [uri|https://github.com/seanhess/hyperbole/blob/main/examples/|]
```

Comment:
In the `0.4` branch which is probably what is live at https://docs.hyperbole.live/simple needs to at minimum change the typo `example` -> `examples` (the links are broken)